tests/unit/core/TEST_BOSTED__3_2.csv

```diff
@@ -0,0 +1,9 @@
+1000000001;3;19850101;19910630;1985;5479;7850
```

Comment:
This file should not be necessary as you have one in  tests/resources/unit_test_data/TEST_BOSTED__3_2.csv